packages/playwright/src/reporters/list.ts

```diff
@@ -245,7 +245,8 @@ class ListReporter extends TerminalReporter {
 
   private _testPrefix(index: string, statusMark: string) {
     const statusMarkLength = stripAnsiEscapes(statusMark).length;
-    return '  ' + statusMark + ' '.repeat(3 - statusMarkLength) + this.screen.colors.dim(index + ' ');
+    const indexLength = Math.ceil(Math.log10(this.totalTestCount));
```

Comment:
I think this does not work for `totalTestCount === 10`, yielding `1`.

Reply:
Oh, good catch! Classic off-by-1 error.

```suggestion
    const indexLength = Math.ceil(Math.log10(this.totalTestCount + 1));
```